src/Centreon/Domain/Filter/FilterService.php

```diff
@@ -124,7 +124,7 @@ public function findFilterByUserId(int $userId, string $pageName, int $filterId)
             return $this->filterRepository->findFilterByUserIdAndId($userId, $pageName, $filterId);
         } catch (\Exception $ex) {
             throw new FilterException(
-                sprintf(_('Error when sarch filter id %d', $filterId)),
+                sprintf(_('Error when searching filter id %d', $filterId)),
```

Comment:
I can't find the translation for this sentence, and the one on line 114.
`https://github.com/centreon/centreon/pull/8854/commits/c169eec5e3a42fc42d7e8133b2f9ee8a684c7358#diff-20ef2c81d54a066c252808d801632343R114`